### ISSUE TITLE
Docs error: sparse_softmax_cross_entropy_with_logits

### ISSUE THREAD

**Issue by piteren** [tensorflow/tensorflow#21435]
There is an error in documentation of sparse_softmax_cross_entropy_with_logits:
https://www.tensorflow.org/api_docs/python/tf/nn/sparse_softmax_cross_entropy_with_logits

there is:
"logits and labels must have the same dtype (either float16, float32, or float64)."

should be:
"logits should have dtype float16, float32, or float64,
 labels should have dtype int32 or int64"

(in Args section of this docs it is written correctly)

**Comment by yongtang** [comment]
@piteren I think you are correct. The labels in `SparseSoftmaxCrossEntropyWithLogits` can only be int32 or int64:

https://github.com/tensorflow/tensorflow/blob/46cf73f0214bc6208295e36650f1a8ffde4abdd7/tensorflow/core/ops/nn_ops.cc#L1092-L1099

Created a PR #21445 for the fix.
